packages/ketcher-polymer-editor-react/src/components/monomerLibrary/RnaBuilder/RnaBuilder.tsx

```diff
@@ -74,7 +74,7 @@ export const RnaBuilder = ({ libraryName }) => {
   return (
     <RnaBuilderContainer>
       <RnaEditor
-        duplicatePreset={duplicatePreset}
+        duplicatePreset={() => duplicatePreset()}
```

Comment:
I am just curious, is`duplicatePreset` and `() => duplicatePreset()` has the same effect here? why isn't `duplicatePreset` better because this funtion is always the same so it would not ben redefined, while `() => duplicatePreset()` is just an anonymous function that would get redefined everytime during render?

Reply:
It's done because to exclude providing "event" as a first parameter of duplicatePreset function, because it waits another optional parameter